docs/.vitepress/config.js

```diff
@@ -54,6 +54,7 @@ export default ({ mode }) => {
         { text: 'Docs', link: '/getting-started.html' },
         { text: 'Widgets', link: '/nodes/widgets.html' },
         { text: 'Contributing', link: '/contributing/' },
+        { text: 'Forum', link: 'https://discourse.nodered.org/tag/dashboard-2' },
```

Comment:
Can we also include the discord for chat?

Reply:
Shouldn't all the documentation really be in one place instead of users having to follow links to other sites...

Reply:
The documentation is all available in the docs, but this links to the Forum for discussion of topics, or to the Discord for those that are operating there instead (as proposed by ZJ)

Reply:
RE: Discord link specifically, I'm a little cautious of over-loading that top-right corner, but can add it if required... doesn't stop this PR though which is constrained just to the "Forum".

Adding a Discord link should be a new PR imo